specification/azsadmin/resource-manager/compute/readme.md

```diff
@@ -64,10 +65,11 @@ csharp:
 input-file:  
     - Microsoft.Compute.Admin/preview/2015-12-01-preview/Compute.json
     - Microsoft.Compute.Admin/preview/2015-12-01-preview/PlatformImages.json
-    - Microsoft.Compute.Admin/preview/2018-02-09/Quotas.json
     - Microsoft.Compute.Admin/preview/2015-12-01-preview/VMExtensions.json
     - Microsoft.Compute.Admin/preview/2018-07-30-preview/Disks.json
     - Microsoft.Compute.Admin/preview/2018-07-30-preview/DiskMigrationJobs.json
+    - Microsoft.Compute.Admin/preview/2020-11-01/Features.json
```

Comment:
Could you please create folder named stable and place the 2020-11-01 related files there? you can make a copy of Compute.json for cross referencing in the stable folder

Reply:
@wmbt96 pls prefer placing it in stable instead of preview

Reply:
you can use the doc at the following location to repro the CI errors locally @wmbt96 
https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/ci-fix.md